Integration test failure

**Describe the bug**
After merging PR #634, the integration tests are failed because of some code conflict.

**To Reproduce**
Run integration tests

**Expected**
Tests should pass.

**Actual behavior**
Tests failed:
```
# github.com/vmware-tanzu/antrea/test/integration/ovs [github.com/vmware-tanzu/antrea/test/integration/ovs.test]
##[error]test/integration/ovs/ofctrl_test.go:561:31: not enough arguments in call to openflow.NewOFBridge
	have (string)
	want (string, string)
##[error]test/integration/ovs/ofctrl_test.go:652:31: not enough arguments in call to openflow.NewOFBridge
	have (string)
	want (string, string)
##[error]test/integration/ovs/ofctrl_test.go:688:31: not enough arguments in call to openflow.NewOFBridge
	have (string)
	want (string, string)
```

**Versions:**
Please provide the following information:
 - Antrea version: 0.6.0

--- a/test/integration/ovs/ofctrl_test.go
+++ b/test/integration/ovs/ofctrl_test.go
@@ -558,7 +558,7 @@ func TestPacketOutIn(t *testing.T) {
 	require.Nil(t, err, fmt.Sprintf("Failed to prepare OVS bridge: %v", err))
 	defer DeleteOVSBridge(br)
 
-	bridge := binding.NewOFBridge(br)
+	bridge := newOFBridge(br)
 	table0 := bridge.CreateTable(0, 1, binding.TableMissActionNext)
 	table1 := bridge.CreateTable(1, 2, binding.TableMissActionNext)
 
@@ -647,9 +647,9 @@ func TestTLVMap(t *testing.T) {
 	br := "br10"
 	err := PrepareOVSBridge(br)
 	require.Nil(t, err, fmt.Sprintf("Failed to prepare OVS bridge: %v", err))
-	//defer DeleteOVSBridge(br)
+	defer DeleteOVSBridge(br)
 
-	bridge := binding.NewOFBridge(br)
+	bridge := newOFBridge(br)
 	table := bridge.CreateTable(0, 1, binding.TableMissActionNext)
 
 	ch := make(chan struct{})
@@ -685,7 +685,7 @@ func TestMoveTunMetadata(t *testing.T) {
 	require.Nil(t, err, fmt.Sprintf("Failed to prepare OVS bridge: %v", err))
 	//defer DeleteOVSBridge(br)
 
-	bridge := binding.NewOFBridge(br)
+	bridge := newOFBridge(br)
 	table := bridge.CreateTable(0, 1, binding.TableMissActionNext)
 
 	err = bridge.Connect(maxRetry, make(chan struct{}))

